src/components/editor-quill/index.js

```diff
@@ -51,17 +47,25 @@ export class EditorQuill extends React.Component {
     this.props.onChange(change)
   }
 
-  // Quill Settings
-  formats() {
-    // Allowed Formats. Docu: https://quilljs.com/docs/formats/
-    return [
-      'header',
-      'bold', 'italic', 'underline',
-      'list', 'indent'
-    ]
+  /**
+   * Quill editor allowed and enabled formats
+   * See https://quilljs.com/docs/formats/
+   *
+   * Example: https://quilljs.com/standalone/full/
+   */
+  getFormats() {
+    const { blockConfig = {} } = this.props
+    // undefined will enable all formats by default
+    return blockConfig.formats || undefined
```

Comment:
doing this, ``{formats: null}`` won't ever be accepted as value, right (will always be "transformed" to ``undefined``)?
```js
console.log('test: ', {formats: null}.formats || undefined)
// test:  undefined
```

Reply:
xD I will remove it ^^